src/index.js

```diff
@@ -1,42 +1,90 @@
+/* eslint-disable no-use-before-define */
 import './style.css';
 
-function showTodo(todo) {
-  const list = document.querySelector('.list');
-  const newlist = document.createElement('li');
-  newlist.classList.add('newlist');
-  newlist.innerHTML = `
-      <input id="${todo.index}" class="check" type="checkbox"/>
-      <span class="text">${todo.description}</span>
-      <button class="delete"><img src="../images/dots.svg"></button>
-    `;
-  list.append(newlist);
+const newTodoForm = document.querySelector('#new-todo-form');
+const todoList = document.querySelector('.todo-list');
+let todos = JSON.parse(localStorage.getItem('todos') || '[]');
+
+function init() {
+  newTodoForm.addEventListener('submit', newTodo);
+  updateList();
+}
+
+function updateTodos(newTodos) {
+  todos = newTodos;
+  localStorage.setItem('todos', JSON.stringify(todos));
+
+  updateList();
+}
```

Comment:
- [ ] Good job so far :+1: Unfortunately as previously mentioned your edit that function doesn't work and we are getting errors. Please try to fix it cause it is part of the project requirement. You can check the screenshot below :arrow_down: 

![Screenshot from 2022-07-16 08-05-38](https://user-images.githubusercontent.com/6615394/179346844-ff384288-69d4-40e4-a7d8-194d6c65ee40.png)


> * [ ]   You should implement the update task function correctly so that someone can edit the task description as well.  When I double click on a task  I am getting an undefined result  error, please fix it
>   ![image](https://user-images.githubusercontent.com/43877689/179251518-9d371307-b9ee-4961-9eef-d3f72f52aac5.png)

Reply:
Please @codecaiine I have already fixed this issue

Reply:
Hi @TuyishimireEric, good to hear But I was getting the same errors as the previous reviewer please make sure that you resolved it so that the next reviewer will approve.  You can see on the screenshot below :point_down:

![Screenshot from 2022-07-16 08-05-38](https://user-images.githubusercontent.com/6615394/179351201-df56e8e1-7126-4d58-b614-205690a566ff.png)